frontend/src/features/properties/selector/search/PropertySelectorSearchContainer.tsx

```diff
@@ -189,7 +188,6 @@ export const featuresToIdentifiedMapProperty = (
           longitude: boundedCenter[0],
           legalDescription: feature?.properties?.LEGAL_DESCRIPTION,
         };
-        property.id = getPropertyIdentifier(property);
```

Comment:
This seems like it could break something. Can you confirm that duplicate detection still works when attempting to add multiple of the same property?

Reply:
Yeah I checked that logic and updated the generation of id's in the table and in the selection as well as tests